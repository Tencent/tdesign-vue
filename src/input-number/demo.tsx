import { defineComponent, ref } from '@vue/composition-api';
import B from './input-number';

export default defineComponent({
  setup() {
    const a = ref(0);

    return <B value={a} onChange={(val) => (a.value = val)} />;
  },
});
