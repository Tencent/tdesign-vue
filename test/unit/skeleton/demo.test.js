import { mount } from '@vue/test-utils';
import demo from '@/examples/skeleton/demos/base.vue';

// unit test for component in examples.
describe('Skeleton', () => {
  it('base demo works fine', () => {
    const wrapper = mount(demo);
    expect(wrapper.element).toMatchSnapshot();
  });
});
