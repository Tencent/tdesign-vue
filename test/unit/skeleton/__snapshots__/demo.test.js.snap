// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton base demo works fine 1`] = `
<div>
  <div
    class="mb-20 t-switch t-size-m t-is-checked"
  >
    <span
      class="t-switch__handle"
    />
    <div
      class="t-switch__content t-size-m"
    />
  </div>
   
  <div>
    <div
      class="t-skeleton"
    >
      <div
        class="t-skeleton__row"
      >
        <div
          class="t-skeleton__col t-skeleton--type-text"
        />
      </div>
      <div
        class="t-skeleton__row"
      >
        <div
          class="t-skeleton__col t-skeleton--type-text"
        />
      </div>
      <div
        class="t-skeleton__row"
      >
        <div
          class="t-skeleton__col t-skeleton--type-text"
        />
      </div>
      <div
        class="t-skeleton__row"
      >
        <div
          class="t-skeleton__col t-skeleton--type-text"
          style="width: 70%;"
        />
      </div>
    </div>
  </div>
</div>
`;
